### ISSUE TITLE
[I2P] "Proxy error: host is down" message after completing tutorial

### ISSUE THREAD

**Issue by Mark137618319** [LukeSmithxyz/landchad#122]
I am having an issue after completing the I2P tutorial. I followed the steps all the way past getting my hostname for my website "i27yllfcqcc3uxxpsf6tb4gphwktxz6k4k3ngmme5zun3js3zkxa.b32.i2p" I am running the i2pd daemon on Windows 10 and changed my computer's proxy settings to match: "Address: 127.0.0.1 | Port: 4444 | Proxy Server Excerpt: 127.0.0.1;localhost" I save changes, and attempt to access the I2P address on Brave. It fails to connect and shows the page: "Proxy error: Host is down. Can't create connection to requested host, it may be down. Please try again later." When I use the "printf "%s.b32.i2p\n" $(head -c 391 /home/i2p/.i2pd/spaceofmark.dat |sha256sum|xxd -r -p | base32 |sed s/=//g | tr A-Z a-z)" command to show my hostname, it gives me "4oymiquy7qobjgx36tejs35zeqt24qpemsnzgtfeswmrw6csxbkq.b32.i2p" a totally different hostname which also throws up the same error. I have tested the i2pd daemon with other I2P websites and they connect without any issues.

**Comment by JacksonTaylorxyz** [comment]
I too get that same hostname. I think the tutorial may be lacking a little bit, but I'm unfamiliar with the technology enough to say anything for sure. I'm trying to figure it out this morning. If I find anything I'll update here.

**Comment by JacksonTaylorxyz** [comment]
Been one looong morning XD I finally got around to messing with this again. I first started running i2pd without the `--daemon` option and instead running it with `--log stdout` and that gave me a lot more to go off of. It seemed like it wasn't able to create my .dat file in the directory. So I changed the permissions to 777 via `chmod 777 ~.i2pd` and then reran it and that allowed the .dat file to be created, not sure why the permissions weren't already correct since my i2p user was the one to create the folder. After that I was able to find my real hostname. I'm still waiting for i2p to create some connections and stuff (I'm still pretty new to the processes and everything, I'm watching a couple videos on it now and getting my local environment setup to be able to view them normally).
